packages/block-library/src/latest-posts/edit.js

```diff
@@ -52,7 +52,11 @@ const USERS_LIST_QUERY = {
 	per_page: -1,
 };
 
-export default function LatestPostsEdit( { attributes, setAttributes } ) {
+export default function LatestPostsEdit( {
+	attributes,
+	setAttributes,
+	className,
```

Comment:
`useBlockProps` is supposed to apply automatically these custom and generated classNames, so I wonder if the issue here is more than the wrapping "div" should be removed and instead the block props applied to the ul

Reply:
Yeah, like I mentioned, I tried passing the className to `useBlockProps` so that it would work correctly, but the class belongs on the `ul`. I didn't think of _removing_ the div, so I've tried that… it also didn't work initially, but clued me in to looking at other blocks, and finally it looks like the `apiVersion` was not updated in #26122, so the className is not being set correctly by `useBlockProps`.

With the fix for `"apiVersion": 2` in the block.json, and moving `blockProps` to the `<ul>` , this seems to be working correctly now.